nss-crystal-cobras/WorkforceManagement
# Db delete training

# Description

This PR closes https://github.com/nss-crystal-cobras/WorkforceManagement/issues/11


## Type of change

Added delete functionality to training programs

- [X ] New feature (non-breaking change which adds functionality)


# Testing Instructions for Change Made

git fetch --all
git checkout db-delete-training
execute bangazonworkforce
click on training programs
try deleting a program
you'll be prompted to see if you really want to delete


# Checklist:

- [X ] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [X ] I have commented my code, particularly in hard-to-understand areas
- [X ] My changes generate no new warnings
- [X ] I have added test instructions that prove my fix is effective or that my feature works
